src/test/java/svc/data/citations/datasources/tyler/TylerCitationDataSourceIntegrationTest.java

```diff
@@ -13,8 +16,8 @@
 import org.springframework.test.context.junit4.SpringRunner;
 
 import svc.Application;
+import svc.models.Citation;
 
-//TODO
 @Ignore
```

Comment:
I see. Maybe instead of real integration, the test can use Pact so we at least verify we're sending the right request: https://github.com/DiUS/pact-jvm